PocketKit/Sources/PocketKit/PocketAppDelegate.swift

```diff
@@ -30,9 +30,7 @@ public class PocketAppDelegate: UIResponder, UIApplicationDelegate {
     }
 
     public func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey : Any]? = nil) -> Bool {
-        if !CommandLine.arguments.contains("disableSentry") {
-            Crashlogger.start(dsn: Keys.shared.sentryDSN)
-        }
+        Crashlogger.start(dsn: Keys.shared.sentryDSN)
```

Comment:
I remember adding this in to avoid sending crash reports while running UI tests and during development. Also, IIRC, the stack traces were sometimes clobbered by sentry during development. I see you've added the conditional compilation bit to set the `debug` option when starting up Sentry. Does that achieve these same results?

Reply:
Ah gotcha, cool I can handle those use cases. We can leverage setting sentry environments here and some other things. I'll look into that.

Reply:
Re-opening. Perhaps it's personal preference, but I did like having Sentry entirely disabled when doing simulator development, for example. I don't feel there's a need to send things up when I'm doing local debugging.

Reply:
@dskuza If there is no valid DSN in your secrets file it will not initialize.

Reply:
I see, so then the suggestion is to just remove it from our local config and really only keep it present when generating a release build?

Reply:
@dskuza correct!